lib/styles/settings/colors/_raw-bodywork.json

```diff
@@ -1 +1 @@
-{"default":[{"id":"_raw-bodywork.scss_white","label":"white","value":"#FFF"}],"theme":[{"id":"_raw-bodywork.scss_theme-50","label":"theme-50","value":"#ECF2F5"},{"id":"_raw-bodywork.scss_theme-100","label":"theme-100","value":"#DCE7ED"},{"id":"_raw-bodywork.scss_theme-200","label":"theme-200","value":"#B6CDD7"},{"id":"_raw-bodywork.scss_theme-300","label":"theme-300","value":"#81B2C8"},{"id":"_raw-bodywork.scss_theme-400","label":"theme-400","value":"#3D8BAF"},{"id":"_raw-bodywork.scss_theme-500","label":"theme-500","value":"#1F7095"},{"id":"_raw-bodywork.scss_theme-600","label":"theme-600","value":"#0E5D81"},{"id":"_raw-bodywork.scss_theme-700","label":"theme-700","value":"#004C6E"},{"id":"_raw-bodywork.scss_theme-800","label":"theme-800","value":"#0A425B"},{"id":"_raw-bodywork.scss_theme-900","label":"theme-900","value":"#183340"}]}
```

Comment:
was it intended that we changed `id` for raw colors in this PR?

Reply:
I changed keys in config files for Typography and Colors. It's only used in for loop in view to have different ID for every item. It has no other consequences.

Reply:
did you see that now it's called `token`?
Also in token file, then `id` starts with `raw` now...

Reply:
Right, fixed.